system/jhtree/jhutil.hpp

```diff
@@ -36,7 +136,7 @@ class CMRUCacheOf : public CInterface//, public IInterface
         virtual void onAdd(void *mapping) { owner.elementAdded((MAPPING *)mapping); TABLE::onAdd(mapping); }
         virtual void onRemove(void *mapping) { owner.elementRemoved((MAPPING *)mapping); TABLE::onRemove(mapping); }
     } table;
-    QueueOf<MAPPING, false> mruList;
```

Comment:
Is QueueOf still used after this change?

Reply:
Yes, used in lots of other places.